prac_08/KivyDemos-master/squaring.kv

```diff
@@ -0,0 +1,26 @@
+# Square Number
+
+BoxLayout:
+    orientation: 'vertical'
+
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(input_number.text)
+        Label:
+            id: output_label
+            font_size: 48
+            color: (1, 0, 1, 1)
```

Comment:
I think this will give out a purple instead of a pink color

Reply:
It is a pink colour to me though, but thank you for the feedback 👍